tools/DocsLinter/DotenvLoader.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Collections.Generic;
+using System.IO;
+
+namespace DocsLinter
+{
+  /// <summary>
+  ///   Helper class for loading the dotenv file.
+  /// </summary>
+  public class DotenvLoader
+  {
+    /// <summary>
+    ///   Helper method that parses the dotenv file.
+    ///   The dotenv file is assumed to be in the cwd.
+    /// </summary>
+    /// <returns>The dotenv file parsed into environment variables.</returns>
+    public static Dictionary<string, string> LoadDotenvFile()
+    {
+      var dotenvContents = new Dictionary<string, string>();
+
+      var dotenvFileContents = File.ReadAllLines(".env");
+
+      foreach (var dotenvRow in dotenvFileContents)
+      {
+        var index = dotenvRow.IndexOf("=");
+
+        if (index < 0)
+        {
+          continue;
+        }
+
+        var key = dotenvRow.Substring(0, index).Trim();
```

Comment:
`dotEnvRow.Split('=').Select(string.Trim)` ? :)

Reply:
You could potentially have a `=` inside a bash variable I think